scala/lambdas/ingest-flow-control/src/main/resources/application.conf

```diff
@@ -0,0 +1,3 @@
+flow-control-queue-table-name = ${QUEUE_DDB_TABLE}
+config-param-name = ${CONFIG_PARAM_NAME}
+step-function-arn=${STEP_FUNCTION_ARN}
```

Comment:
Could you change the name to `INGEST_SFN_ARN` so that it is consistent with the parsed court document handler?